inc/auth.class.php

```diff
@@ -273,6 +275,8 @@ function connection_ldap($ldap_method, $login, $password) {
             //Use is not present anymore in the directory!
             if ($dn == '') {
                $this->user_deleted_ldap = true;
+            } else {
+               $this->user_found = true;
             }
```

Comment:
Please confirm: 
We think if the user is not in a LDAP, DN is empty and user is set to be deleted.
If this happens in the last LDAP the user will be always deleted, even if exists in a previous LDAP server. If there is a better way to break the loop, great. Otherwise, we have tested it and this works.